docs/website/generator-errors.md

```diff
@@ -261,6 +261,8 @@ interface CICustomFilter  {
 }
 ```
 
+### <a name='BI1120'/>BI1120: Unsupported platform string: *. Platform strings should contain the version of the platform supported. Examples are: maccatalyst15.0, mac10.9, ios8.0, tvos15.0
```

Comment:
I think this should go below the next paragraph ("If the property is inlined ..."), because otherwise that paragraph will be out of place.